src/libopenrave/kinbody.cpp

```diff
@@ -5984,11 +5987,12 @@ void KinBody::ExtractInfo(KinBodyInfo& info)
     info._vGrabbedInfos.resize(0);
     GetGrabbedInfo(info._vGrabbedInfos);
 
-    info._transform = GetTransform();
+    // GetTransform() = info._transform * _baseLinkRelativeTransform
+    info._transform = GetTransform() * _baseLinkRelativeTransform.inverse();
```

Comment:
err... doesn't make sense, shouldn't `info._transform` should be the body frame that is returned by GetTransform()